assets/xml/objects/object_tro.xml

```diff
@@ -1,14 +1,18 @@
 ﻿<Root>
     <File Name="object_tro" Segment="6">
-        <Animation Name="object_tro_Anim_0000A0" Offset="0xA0" />
-        <DList Name="object_tro_DL_0006B0" Offset="0x6B0" />
-        <Texture Name="object_tro_TLUT_000AE0" OutName="tlut_000AE0" Format="rgba16" Width="16" Height="16" Offset="0xAE0" />
-        <Texture Name="object_tro_TLUT_000CE0" OutName="tlut_000CE0" Format="rgba16" Width="16" Height="16" Offset="0xCE0" />
-        <Texture Name="object_tro_Tex_000EE0" OutName="tex_000EE0" Format="rgba16" Width="2" Height="8" Offset="0xEE0" />
-        <Texture Name="object_tro_Tex_000F00" OutName="tex_000F00" Format="ci8" Width="16" Height="8" Offset="0xF00" />
-        <Texture Name="object_tro_Tex_000F80" OutName="tex_000F80" Format="rgba16" Width="16" Height="8" Offset="0xF80" />
-        <Texture Name="object_tro_Tex_001080" OutName="tex_001080" Format="rgba16" Width="8" Height="8" Offset="0x1080" />
-        <Texture Name="object_tro_Tex_001100" OutName="tex_001100" Format="ci8" Width="8" Height="8" Offset="0x1100" />
-        <!-- <Blob Name="object_tro_Blob_001140" Size="0x1820" Offset="0x1140" /> -->
+        <Animation Name="gKioskKoumeHeadMoving" Offset="0xA0" />
+        <DList Name="gKioskKoumeDL" Offset="0x6B0" />
+        <Texture Name="gKioskKoumeNoseTLUT" OutName="kiosk_koume_nose_tlut" Format="rgba16" Width="16" Height="16" Offset="0xAE0" />
+        <Texture Name="gKioskKoumeNostrilsTLUT" OutName="kiosk_koume_nostrils_tlut" Format="rgba16" Width="16" Height="16" Offset="0xCE0" />
+        <Texture Name="gKioskKoumeShadowedSkinTex" OutName="kiosk_koume_shadowed_skin" Format="rgba16" Width="2" Height="8" Offset="0xEE0" />
+        <Texture Name="gKioskKoumeNoseTex" OutName="kiosk_koume_nose" Format="ci8" Width="16" Height="8" Offset="0xF00" />
+        <Texture Name="gKioskKoumeJewelHolderTex" OutName="kiosk_koume_jewel_holder" Format="rgba16" Width="16" Height="8" Offset="0xF80" />
+        <Texture Name="gKioskKoumeJewelTex" OutName="kiosk_koume_jewel" Format="rgba16" Width="8" Height="8" Offset="0x1080" />
+        <Texture Name="gKioskKoumeNostrilsTex" OutName="kiosk_koume_nostrils" Format="ci8" Width="8" Height="8" Offset="0x1100" />
+        <Texture Name="gKioskKoumeEyeOpenTex" OutName="kiosk_koume_eye_open" Format="rgba16" Width="32" Height="32" Offset="0x1140"/>
+        <Texture Name="gKioskKoumeEyeHalfTex" OutName="kiosk_koume_eye_half" Format="rgba16" Width="32" Height="32" Offset="0x1940"/>
+        <Texture Name="gKioskKoumeEyeClosedTex" OutName="kiosk_koume_eye_closed" Format="rgba16" Width="32" Height="32" Offset="0x2140"/>
+        <Limb Name="gKioskKoumeLimb" Type="Standard" EnumName="KIOSK_KOUME_LIMB_1" Offset="0x2940" />
```

Comment:
```suggestion
        <Limb Name="gKioskKoumeHeadLimb" Type="Standard" EnumName="KIOSK_KOUME_LIMB_HEAD" Offset="0x2940" />
```
tbh, I would be a little more specific (especially since you named the anim `gKioskKoumeHeadMoving`).